osquery/tables/yara/yara.cpp

```diff
@@ -21,6 +27,18 @@
 #include <yara.h>
 
 namespace osquery {
+
+// After a large scan of many files, the memory allocation could be
+// substantial.  free() may not return it to operating system, but
+// rather keep it around in anticipation that app will reallocate.
+// Call malloc_trim() on linux to try to convince it to release.
+#ifdef LINUX
+FLAG(bool,
+     yara_malloc_trim,
+     true,
+     "Call malloc_trim() after yara scans (linux)");
```

Comment:
Nitpick, `Call malloc_trim after YARA scans (Linux only)`, YARA is usually written in caps right?